src/checks.rs

```diff
@@ -1615,6 +1618,7 @@ impl CheckCode {
             CheckCode::T201 => CheckCategory::Flake8Print,
             CheckCode::T203 => CheckCategory::Flake8Print,
             CheckCode::UP001 => CheckCategory::Pyupgrade,
+            CheckCode::UP002 => CheckCategory::Pyupgrade,
```

Comment:
Sorry for the inconvenience, but do you mind changing this to `UP019`? `UP002` used to be mapped to a different check, which we had to remove (`pyupgrade` removed it too, it was "wrong"), and so using `UP002` would do the wrong thing here if users upgrade from old to new versions.

Reply:
I can definitely do that, how do I know which code to use, I tried looking on pyupgrade but didn't see (for future reference)?

Reply:
They’re just made up, since pyupgrade doesn’t have codes. So we use the next available slot each time (skipping UP002).

Reply:
Perfect. Got this fixed!